### PR TITLE
Slack - add support in credential store

### PR DESCRIPTION
**Pull request by mmhw**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
Relates: [link to the issue](https://jira-hq.paloaltonetworks.local/browse/CIAC-4767)

## Description
Change the type of credentials, from type 4 to type 9.

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Review by bziser — APPROVED** [review]
Looks good

Let's wait for @amshamah419 also review :)

**Comment by amshamah419** [comment]
@mmhw Looks good to me. Just pull from master before merging (I bumped the pack version on another merge).

**Pull request merged**
